lib/3scale_toolbox/commands/remote_command.rb

```diff
@@ -19,9 +19,16 @@ def self.command
           end
         end
 
+        # list remotes
         def run
-          # list remotes
-          puts "list remotes"
+          if ThreeScaleToolbox.configuration.remotes.empty?
+            puts 'Emtpy remote list.'
+            exit 0
```

Comment:
Calling `exit 0` from commands will make it quite hard to test. IMO we should rely on other mechanisms rather than just directly exiting.

Reply:
Yeah, it will be abstracted. Console IO will be abstracted as well  to be easy to test.